gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -197,22 +200,35 @@ private void createPipelineTabs(TabFolder folder, List<API.Pipeline> pipelines)
         for (API.Stage stage : stages) {
           TabItem item = createStandardTabItem(folder, stage.getDebugName());
 
-          Group stageGroup = createGroup(folder, stage.getStageName());
+          Composite stageGroup = createComposite(folder, new GridLayout());
+
+          FillLayout nameLayout = new FillLayout(SWT.VERTICAL);
+          nameLayout.marginHeight = 5;
+          Composite nameComposite = withLayoutData( createComposite(stageGroup, nameLayout),
+              new GridData(SWT.BEGINNING, SWT.BEGINNING, false, false));
+
+          Label stageName = createBoldLabel(nameComposite, stage.getStageName() + " (" + stage.getDebugName() + ")");
+          FontData[] fd = stageName.getFont().getFontData();
+          fd[0].setHeight(14);
+          stageName.setFont( new Font(getDisplay(), fd[0]));
```

Comment:
Fonts are resources that need to be life-time managed. (i.e. at some point you need to call `.dispose()` on them. Add it to the theme, so this is managed for you.

Reply:
Should I be use bigBoldFont() from Theme.java?